### PR TITLE
start.json

### PR DESCRIPTION
**Pull request by princetoday**
Basic Information = Specifies the monster's name and total health.
Default States= Defines the monster's default states, including arriving (with animations and sounds), idle (with stats like armor, attack, defense, speed, and abilities), and a dying_state.
Phased States = Details how the monster transitions through three health phases:
1.0 Phase (100% Health) = Custom arrival animations and sounds while using default idle stats.
0.5 Phase (50% Health)=  Introduces new arrival animations and an additional armor value.
0.25 Phase (25% Health)= Further customizes the arrival and significantly enhances attack and armor values, indicating the monster's heightened danger.

### CODE REVIEW

**Comment by Chovin** [comment]
sorry, let me rephrase, the #11 issue is to make phases work in the [Invasion cog](https://github.com/Chovin/Dumb-Cogs/tree/main/invasion). I was just giving you an example format that could work for phases during our conversation. I'm pretty sure the only thing you'd need to alter is [`enemy.py`](https://github.com/Chovin/Dumb-Cogs/blob/main/invasion/enemy.py) to get phases working. Once you do get it working, you can test that they work by using `[p]provoke` after setting up a test monster in `invasion/data/enemies/`. please do make sure you follow the format described in [here](https://github.com/Chovin/Dumb-Cogs/blob/main/CONTRIBUTING.md#i-want-to-add-more-monsters-to-the-invasion-cog) though (besides your changes for phases). For example, there's no `abilities` field or `sound` field

**Review by Chovin — CHANGES_REQUESTED** [review]
Please see previous comment. This issue isn't for creating a new monster that has phases, it's to add the capability of having enemies with phases.

File: invasion/enemy.py [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
 import asyncio
 import math
 import re
 import os
 
-
 ARRIVING_STATE = "arriving"
 WIN_STATE = "dying"
-LOSE_STATE = "attacking"
+def new_func():
```
Comment (Chovin) [inline invasion/enemy.py:12]: this function isn't needed, please remove this ❤️

File: invasion/enemy.py [hunk 2]
```diff
@@ -50,36 +50,24 @@ def __init__(self, path: str, min_enrage_mult: float, max_enrage_mult: float, en
...
-            "Mythic",
-            "Ancient",
-            "Colossal",
-            "Unstoppable",
-            "Apocalyptic",
-            "Eternal",
-            "Divine"
+            "Angry", "Enraged", "Formidable", "Rampaging", "Ferocious",
+            "Epic", "Mythic", "Ancient", "Colossal", "Unstoppable",
+            "Apocalyptic", "Eternal", "Divine"
```
Comment (Chovin) [inline invasion/enemy.py:56]: let's keep these each on a single line like they were before. makes them easier to read and maintain

File: invasion/enemy.py [hunk 3]
```diff
@@ -158,7 +146,6 @@ def title_prefix(self):
         px = len(self._title_prefixes) - 1
 
         return self._title_prefixes[
-            # map value from between mn and mx (exclusive) to between pn and px (inclusive)
```
Comment (Chovin) [inline invasion/enemy.py:161]: please put this comment back

File: invasion/enemy.py [hunk 4]
```diff
@@ -182,60 +169,70 @@ def hurt(self, player: Member, dmg_type: str, damage: int):
             return
         is_bomb = dmg_type == BOMB_DMG_TYPE
         pid = player.id
-        # dmg as an arg for purchasable bombs
```
Comment (Chovin) [inline invasion/enemy.py:185]: please put these comments back

File: invasion/enemy.py [hunk 5]
```diff
@@ -182,60 +169,70 @@ def hurt(self, player: Member, dmg_type: str, damage: int):
...
         self.attacked_by[player.id] = self.attacked_by.get(player.id, 0) + min(self.health, dmg)
         if not is_bomb:
-            # make future hits weaker
-            self.hurt_mult[pid] = self.hurt_mult.get(pid, 1) * .75
+            self.hurt_mult[pid] = self.hurt_mult.get(pid, 1) * 0.75
         else:
             self.bombed_by[player.id] = self.bombed_by.get(player.id, 0) + 1
         return dmg
 
-    def attack(self):
```
Comment (Chovin) [inline invasion/enemy.py:203]: please put this function back

File: invasion/enemy.py [hunk 6]
```diff
@@ -182,60 +169,70 @@ def hurt(self, player: Member, dmg_type: str, damage: int):
...
     def tick(self):
         if self.health <= 0:
             self.health = 0
             self.state = WIN_STATE
             return
+
+        # Health-based state transitions
+        health_percentage = self.health_percentage
         
+        if health_percentage > 0.75:
```
Comment (Chovin) [inline invasion/enemy.py:196]: the phases should be based on the configuration done in the `stats.json` file for each monster. please don't hard-code a single monster's configuration into the code

**Review by Chovin — CHANGES_REQUESTED** [review]
please fix the commented sections.
also, it's alright since it's your first contribution, but in general you should only include the changes that implement the feature/fix in your PR. so for example all the formatting changes shouldn't be included in your commits

**Comment by princetoday** [comment]
Ok I do again today
